### PR TITLE
Use precompiled ffmpeg

### PR DESCRIPTION
**Pull request by DominikWolek**

### CODE REVIEW

**Review by Noarkhh — CHANGES_REQUESTED** [review]
Like in `membrane_ffmpeg_video_filter_plugin`, readme and version

File: mix.exs [hunk 1]
```diff
@@ -37,7 +37,7 @@ defmodule Membrane.FFmpeg.SWScale.Mixfile do
       {:membrane_core, "~> 0.12.8"},
       {:membrane_raw_video_format, "~> 0.3.0"},
       {:membrane_common_c, "~> 0.15.0"},
-      {:unifex, "~> 1.0"},
+      {:bundlex, "~> 1.2"},
```
Comment (Noarkhh) [inline mix.exs:40]:
```suggestion
      {:bundlex, "~> 1.2.0"},
```

Comment (DominikWolek) [reply mix.exs:40]: Why?

Comment (Noarkhh) [reply mix.exs:40]: my mistake, I got too used to working with `0.x.y` versions

**Review by Noarkhh — APPROVED** [review]

**Pull request merged**